pai-management/k8sPaiLibrary/maintainlib/deploy.py

```diff
@@ -118,8 +118,9 @@ def create_kube_proxy(self):
         generated_data = common.generate_from_template_dict(template_data, dict_map)
 
         common.write_generated_file(generated_data, "kube-proxy.yaml")
-        common.execute_shell(
+        common.execute_shell_retry(
             "kubectl apply --overwrite=true -f kube-proxy.yaml",
+            "kubectl delete -f kube-proxy.yaml",
```

Comment:
A little confuse about this, if we failed to create a daemonset and the the object may not exists on k8s. Why should we still delete it?

Reply:
I am considering when Daemonset is created but no pod in it. 
Remind by you, I am confused that whether this condition will raise "kube-proxy created failed" error?